packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue

```diff
@@ -407,13 +405,17 @@ ul {
 }
 
 /* TODO: Improve this pattern later same in (tera-model-input) */
+
+.notebook-section {
+	background-color: var(--surface-disabled);
+	border-right: 1px solid var(--surface-border-dark);
+	padding: var(--gap);
+}
```

Comment:
This rule is being repeated. See about moving the rule to `tera-drilldown-section` and I believe the class would still be read from here

Reply:
Done!